### PR TITLE
[RW-6749][risk=no] Increase CircleCI e2e test parallelism

### PR DESCRIPTION
**Pull request by aweng98**
looks like running 3 tests concurrently on a single CircleCI node is okay without any issue.

e2e tests current runtime is ~22 min.
Screen Shot 2021-07-09 at 2 11 52 PM

after change.
Screen Shot 2021-07-09 at 2 11 24 PM

An increase in api peak load on appengine could lead to more `request has timed out ...`.

### CODE REVIEW

**Comment by aweng98** [comment]
no rush.

File: e2e/package.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "test-local-devup:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_HEADLESS=false DEBUG=true node --inspect-brk node_modules/.bin/jest --maxWorkers=1",
     "test-staging": "cross-env WORKBENCH_ENV=staging yarn _test --maxWorkers=3",
     "test-staging:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_HEADLESS=false DEBUG=true yarn _test --maxWorkers=1",
-    "test:ci": "yarn _test --ci --maxWorkers=2 --forceExit",
-    "test:ci:debug": "yarn _test --ci --debug --maxWorkers=1 --bail 1",
+    "test:ci": "yarn _test --ci --maxWorkers=3 --forceExit",
+    "test:ci:debug": "yarn _test --ci --debug --maxWorkers=3",
```
Comment (calbach) [inline e2e/package.json:29]: Why change the bail setting here?

**Review by calbach — APPROVED** [review]

File: e2e/package.json (hunk 1 above)
Comment (aweng98) [reply e2e/package.json:29]: I think we still want to run all test suites even if there is one failed suite. I don't feel strongly. Should I add it back?

Comment (calbach) [reply e2e/package.json:29]: Sounds fine, I was just curious why it was changing as part of this PR.

**Pull request merged**